packages/web3-core-method/src/index.js

```diff
@@ -618,22 +618,34 @@ Method.prototype.buildCall = function () {
 
         // CALLBACK function
         var sendTxCallback = function (err, result) {
-            if (method.handleRevert && !err && isCall && (method.isRevertReasonString(result) && method.abiCoder)) {
-                var reason = method.abiCoder.decodeParameter('string', '0x' + result.substring(10));
-                var signature = 'Error(String)';
-
-                utils._fireError(
-                    errors.RevertInstructionError(reason, signature),
-                    defer.eventEmitter,
-                    defer.reject,
-                    payload.callback,
-                    {
-                        reason: reason,
-                        signature: signature
-                    }
-                );
+            if (method.handleRevert && isCall && method.abiCoder) {
+                var reasonData;
+
+                // Ganache / Geth <= 1.9.13 return the reason data as a successful eth_call response
```

Comment:
great, what do you think about adding something like this to help document for historical reasons:
`// Geth 1.9.14 has revert reason missing (https://github.com/ethereum/web3.js/issues/3520)`

Might save someone some time somewhere down the line 😄

Reply:
c905c3